config/config.go

```diff
@@ -3,21 +3,23 @@ package config
 import "golbat/geo"
 
 type configDefinition struct {
-	Port      int        `koanf:"port"`
-	Webhooks  []webhook  `koanf:"webhooks"`
-	Database  database   `koanf:"database"`
-	Stats     bool       `koanf:"stats"`
-	Logging   logging    `koanf:"logging"`
-	Sentry    sentry     `koanf:"sentry"`
-	Pyroscope pyroscope  `koanf:"pyroscope"`
-	InMemory  bool       `koanf:"in_memory"`
-	Cleanup   cleanup    `koanf:"cleanup"`
-	RawBearer string     `koanf:"raw_bearer"`
-	ApiSecret string     `koanf:"api_secret"`
-	Pvp       pvp        `koanf:"pvp"`
-	Koji      koji       `koanf:"koji"`
-	Tuning    tuning     `koanf:"tuning"`
-	ScanRules []scanRule `koanf:"scan_rules"`
+	Port              int        `koanf:"port"`
+	Webhooks          []webhook  `koanf:"webhooks"`
+	Database          database   `koanf:"database"`
+	Stats             bool       `koanf:"stats"`
+	Logging           logging    `koanf:"logging"`
+	Sentry            sentry     `koanf:"sentry"`
+	Pyroscope         pyroscope  `koanf:"pyroscope"`
+	LegacyInMemory    bool       `koanf:"in_memory"`
```

Comment:
You might want to rename the config too, according to your description. 
```suggestion
	LegacyInMemory    bool       `koanf:"legacy_in_memory"`
```

Reply:
It's actually removed entirely now... but I have left this piece behind